monitoring-compose.yml

```
@@ -9,23 +9,23 @@ services:
   zipkin:  # Zipkin
     image: openzipkin/zipkin
     container_name: zipkin
     ports:
       - "9411:9411"
     networks:
-      - monitoring-network
+      - auction-network
 
 networks:
-  monitoring-network:
+  auction-network:
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)